scripts/globals/roe.lua

```diff
@@ -243,10 +249,7 @@ local function completeRecord(player, record)
         type(rewards["accolades"]) == "number"
     then
         local bonusAccoladeRate = 1.0
-
-        if record ~= 5 then -- Do not grant a bonus for All for One
-            bonusAccoladeRate = bonusAccoladeRate + ((player:getUnityRank() - 1) * 0.05)
-        end
+        bonusAccoladeRate = bonusAccoladeRate + (player:getUnityRank() or 0) * 0.05
 
         player:addCurrency("unity_accolades", math.floor(rewards["accolades"] * bonusAccoladeRate), CAP_CURRENCY_ACCOLADES)
         player:messageBasic(xi.msg.basic.ROE_RECEIVED_ACCOLADES, rewards["accolades"], player:getCurrency("unity_accolades"))
```

Comment:
Should this Accolades received message be showing the actual amount you received, after the bonus calculation? Or the pre-bonus amount as it does now? :thinking:

Reply:
I'll agree that it should be showing full; however, Record ID 5 will not receive this bonus (static 1000 value).  Also, Rank 1 unity receives no bonus, and that logic is removed in this PR.

Reply:
Actually, I checked that. It appears as though the "subtract 1 from unity rank" is already being done on the `getUnityRank()` end. So it was in fact subtracting 5% for being Unity rank 1 before. This was likely never caught because the message always showed the flat reward.

Reply:
Its a little confusing, but unity leader is 1-indexed with "None" being 0.  The rank values in the DB are also reflective of their actual ranking (not 0-index)

Reply:
I see. Reinstated the -1.